pyexodus/core.py

```diff
@@ -459,6 +471,41 @@ def put_element_variable_values(self, blockId, name, step, values):
 
         self._f.variables[variable_name][step - 1] = values
 
+    def get_element_variable_values(self, blockId, name, step):
+        """
+        Get values from element block id and variable name at step.
+
+        :type blockId: int
+        :param blockId: The block id.
+        :type name: str
+        :param name: The name of the variable.
+        :type step: int
+        :param step: The time step at which to put the values.
+        Return values: The actual values.
+        """
+        assert step > 0, "Step must be larger than 0."
+        # XXX: Currently the time axis is not unlimited due to a limitation
+        # in h5netcdf - thus no new time steps can be created after the
+        # initialization.
+        assert step <= self._f.dimensions["time_step"]
+
+        num_elem_name = "num_el_in_blk%i" % blockId
+        assert num_elem_name in self._f.dimensions, \
+            "Block id %i not found." % blockId
+
+        # 1-based indexing!
+        idx = self.get_element_variable_names().index(name) + 1
+
+        variable_name = "vals_elem_var%ieb%i" % (idx, blockId)
+
+        # If it does not exist, create it.
+        if variable_name not in self._f.variables:
```

Comment:
The if branch is also not executed in the tests and codecov thus warns of reduced coverage.